pep/app/pods/components/document/text/component.ts

```diff
@@ -607,4 +651,13 @@ export default class DocumentText extends Component<DocumentTextArgs> {
         });
         return results.documents?.responseSet?.[0].document;
     }
+
+    /**
+     * Turn off page tracking so even if the user happens to scroll before the component is destroyed, we dont update the page
+     *
+     * @memberof DocumentText
+     */
+    willDestroy(): void {
+        this.pageTracking = false;
+    }
```

Comment:
this is fine, but using a concurrency task would've handled this for you too, w/o the need for the additional `pageTracking` state and this extra boilerplate.